apl-core/src/main/java/com/apollocurrency/aplwallet/apl/core/http/get/GetAllExchanges.java

```diff
@@ -50,7 +50,7 @@ public JSONStreamAware processRequest(HttpServletRequest req) throws AplExceptio
 
         JSONObject response = new JSONObject();
         JSONArray exchanges = new JSONArray();
-        try (DbIterator<Exchange> exchangeIterator = Exchange.getAllExchanges(firstIndex, lastIndex)) {
+        try (DbIterator<Exchange> exchangeIterator = exchangeService.getAllExchanges(firstIndex, lastIndex)) {
```

Comment:
Not sure that we need both implementations here:  `getAllExchanges` and `getAllExchangesStream`
We can rewrite this code with streams, as simple as that:
        `JSONObject response = new JSONObject();
        JSONArray ...= exchangeService.getAllExchangesStream(firstIndex, lastIndex)
            .takeWhile(trade -> trade.getTimestamp() >= timestamp)
            .map(e-> SONData.exchange(exchange, includeCurrencyInfo))
            .collect(CollectorUtils.jsonCollector());`
What's your opinion?

Reply:
In general I don't like idea to fetch 'everything' by firstIndex, lastIndex and filtering result by 'timestamp'. I suppose we have to run query by using 'timestamp' in SQL.
But I didn't touch 'old REST API' code because it's 'out of scope' that task in general.